services/director-v2/tests/unit/conftest.py

```diff
@@ -95,6 +95,11 @@ def request_scheme() -> str:
     return "http"
 
 
+@pytest.fixture
+def can_save() -> bool:
+    return True
```

Comment:
why not test with True and False?

```python

@pytest.fixture(params=[True, False])
def can_save(request: FixtureRequest) -> bool:
    return request.param

```

Reply:
It is not necessary to trigger these tests twice to figure out if nodeports can save the data.